src/node.rs

```diff
@@ -3322,4 +3369,25 @@ mod tests {
             .expect("failed getting filter changes");
         assert_eq!(0, result.len());
     }
+
+    #[tokio::test]
```

Comment:
Thoughts about an e2e test also?

```typescript
describe("eth_accounts", function () {
  it("Should return rich accounts", async function () {
    // Arrange
    const richAccounts = RichAccounts.map(ra => ra.Account);

    // Act
    const response: string[] = await provider.send("eth_accounts", []);

    // Assert
    expect(response).to.eq(richAccounts);
  });
});
```

This would require a change to the `RichAccounts` `const`, which it should probably be an array anyways 🤷 :
```typescript
export const RichAccounts = [
  {
    Account: "0x36615Cf349d7F6344891B1e7CA7C72883F5dc049",
    PrivateKey: "0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110",
  },
  {
    Account: "0xa61464658AfeAf65CccaaFD3a512b69A83B77618",
    PrivateKey: "0xac1e735be8536c6534bb4f17f06f6afc73b2b5ba84ac2cfb12f7461b20c0bbe3",
  },
  {
    Account: "0x0D43eB5B8a47bA8900d84AA36656c92024e9772e",
    PrivateKey: "0xd293c684d884d56f8d6abd64fc76757d3664904e309a0645baf8522ab6366d9e",
  },
  {
    Account: "0xA13c10C0D5bd6f79041B9835c63f91de35A15883",
    PrivateKey: "0x850683b40d4a740aa6e745f889a6fdc8327be76e122f5aba645a5b02d0248db8",
  },
  {
    Account: "0x8002cD98Cfb563492A6fB3E7C8243b7B9Ad4cc92",
    PrivateKey: "0xf12e28c0eb1ef4ff90478f6805b68d63737b7f33abfa091601140805da450d93",
  },
  {
    Account: "0x4F9133D1d3F50011A6859807C837bdCB31Aaab13",
    PrivateKey: "0xe667e57a9b8aaa6709e51ff7d093f1c5b73b63f9987e4ab4aa9a5c699e024ee8",
  },
  {
    Account: "0xbd29A1B981925B94eEc5c4F1125AF02a2Ec4d1cA",
    PrivateKey: "0x28a574ab2de8a00364d5dd4b07c4f2f574ef7fcc2a86a197f65abaec836d1959",
  },
  {
    Account: "0xedB6F5B4aab3dD95C7806Af42881FF12BE7e9daa",
    PrivateKey: "0x74d8b3a188f7260f67698eb44da07397a298df5427df681ef68c45b34b61f998",
  },
  {
    Account: "0xe706e60ab5Dc512C36A4646D719b889F398cbBcB",
    PrivateKey: "0xbe79721778b48bcc679b78edac0ce48306a8578186ffcb9f2ee455ae6efeace1",
  },
  {
    Account: "0xE90E12261CCb0F3F7976Ae611A29e84a6A85f424",
    PrivateKey: "0x3eb15da85647edd9a1159a4a13b9e7c56877c4eb33f614546d4db06a51868b1c",
  },
 ] as const;
```